tests/FsCheck.Test/Gen.fs

```diff
@@ -119,6 +119,13 @@ module Gen =
         Gen.listOfLength length (Gen.constant v)
         |> sample1
         |> ((=) (List.init length (fun _ -> v)))
+
+    [<Property>]
+    let Shuffle (xs:Set<int>) =
+        not xs.IsEmpty ==> lazy
+        Gen.shuffle xs
+        |> sample 10
+        |> List.exists ((<>) (Set.toSeq xs))
```

Comment:
The only property of `shuffle` that I can think of:

- `List.sort xs` should be equal to `List.sort shuffledXs` (I'm suggesting `List` here, because lists have structural equality, so they are easy to compare.)

We'd also _like_ to be able to state that `xs` should be different from the shuffled sequence, but although random, this property can't be guaranteed - particularly not for small lists.